frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```diff
@@ -299,6 +299,7 @@
   "Remove Subscription": "Remove Subscription",
   "View ClusterServiceVersion...": "View ClusterServiceVersion...",
   "Channel": "Channel",
+  "Approval strategy": "Approval strategy",
```

Comment:
Hm, I'm surprised this is still in the messages. It looks like you got all the occurrences I could find.

```
❯ rg 'Approval strategy' -tts -tjs
packages/operator-lifecycle-manager/src/components/subscription.tsx
265:        title: t('olm~Approval strategy'),

packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx
732:                  <label className="co-required">{t('olm~Approval strategy')}</label>

packages/operator-lifecycle-manager/src/components/subscription.spec.tsx
296:      'olm~Approval strategy',
```

@florkbr Did you run `yarn i18n`?

Reply:
@spadgett I did run `yarn i18n`. Note there is still one place I think the string is used here: https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/subscription.tsx#L265

I'm not sure if that needs to be updated as well. I could not find where it is rendered in the UI.

Reply:
Ah ha, you can see that on the Search page here:

http://localhost:9000/search/ns/openshift-operators?kind=operators.coreos.com%7Ev1alpha1%7ESubscription

<img width="1554" alt="Screen Shot 2021-04-20 at 3 16 47 PM" src="https://user-images.githubusercontent.com/1167259/115451663-7bb86280-a1eb-11eb-9e72-2e0aacc3dd54.png">

I'd update that instance as well so we're consistent.

Reply:
@spadgett done!

Reply:
<img width="1791" alt="Screen Shot 2021-04-21 at 11 01 37 AM" src="https://user-images.githubusercontent.com/21317056/115575937-0737ff00-a291-11eb-9946-bbdf29a611b1.png">